examples/clone.c

```diff
@@ -23,11 +23,11 @@ static void print_progress(const progress_data *pd)
 
 	if (pd->fetch_progress.total_objects &&
 		pd->fetch_progress.received_objects == pd->fetch_progress.total_objects) {
-		printf("Resolving deltas %d/%d\r",
+		printf("Resolving deltas %u/%u\r",
 		       pd->fetch_progress.indexed_deltas,
 		       pd->fetch_progress.total_deltas);
 	} else {
-		printf("net %3d%% (%4"PRIuZ" kb, %5d/%5d)  /  idx %3d%% (%5d/%5d)  /  chk %3d%% (%4" PRIuZ "/%4" PRIuZ ") %s\n",
+		printf("net %3d%% (%4d kb, %5u/%5u)  /  idx %3d%% (%5u/%5u)  /  chk %3d%% (%4" PRIuZ "/%4" PRIuZ ") %s\n",
```

Comment:
The `%u` conversions are the right thing to do, but the "PRIuZ" needs to stay as `kbytes` is of type `size_t`

Reply:
I think that I missed that when I reworked original after a source change. Will fix this `residual`.